pythonExecSource/docs/Usage.md

```diff
@@ -32,6 +32,14 @@ Note: The term _extension source_ is an older term, but it is still evident in t
 
 ## Python Execution Connector Operation
 
```

Comment:
Add _warning_ that you're executing code so code executions can happen.  In this case, it's "real code".  Other connectors can have an effect on the things to which they connect (that's there purpose), but it was suggested that we call this out.

Fully sandboxing Python seems to be well-nigh impossible, in general.